modules/understanding-upgrade-channels.adoc

```diff
@@ -117,30 +94,29 @@ The presence of an update recommendation in the `stable-4` channel at any point
 endif::openshift-origin[]
 
 ifndef::openshift-origin[]
-[discrete]
-== Fast and stable channel use and strategies
+
+= Fast and stable channel use and strategies
 
 The `fast-{product-version}` and `stable-{product-version}` channels present a choice between receiving general availability releases as soon as they are available or allowing Red Hat to control the rollout of those updates. If issues are detected during rollout or at a later time, upgrades to that version might be blocked in both the `fast-{product-version}` and `stable-{product-version}` channels, and a new version might be introduced that becomes the new preferred upgrade target.
 
 Customers can improve this process by configuring pre-production systems on the `fast-{product-version}` channel, configuring production systems on the `stable-{product-version}` channel, and participating in the Red Hat connected customer program. Red Hat uses this program to observe the impact of updates on your specific hardware and software configurations. Future releases might improve or alter the pace at which updates move from the `fast-{product-version}` to the `stable-{product-version}` channel.
 endif::openshift-origin[]
 
-[discrete]
-== Restricted network clusters
+= Restricted network clusters
 
 If you manage the container images for your {product-title} clusters yourself, you must consult the Red Hat errata that is associated with product releases and note any comments that impact upgrades. During upgrade, the user interface might warn you about switching between these versions, so you must ensure that you selected an appropriate version before you bypass those warnings.
 
 ifndef::openshift-origin[]
-[discrete]
-== Switching between channels
+
+= Switching between channels
 
 A channel can be switched from the web console or through the `adm upgrade channel` command:
```

Comment:
It's saying `A channel can be switched from the web console`, but we don't document how to change it from web console.